src/components/cc-dashboard/index.cjsx

```diff
@@ -90,6 +91,26 @@ DashboardChapter = React.createClass
       {chapter}
       {sections}
     </div>
+
+DashboardBookLinks = React.createClass
+  goToPdf: ->
+    course = CourseStore.get(@props.courseId)
+    window.open(course.book_pdf_url, 'cc-pdf-link')
```

Comment:
Should this get analytics added to it? Is there an `<OpenInNewTab>` component?

/cc @nathanstitt